pydicom/tests/test_filewriter.py

```diff
@@ -1090,6 +1090,26 @@ def test_correct_ambiguous_raw_data_element(self):
         assert out.VR == 'US'
         assert out.value == 0xfffe
 
+    def test_empty_value(self):
+        """Regression test for #1193: empty value raises exception."""
+        ds = Dataset()
+        elem = RawDataElement(0x00280106, 'US or SS', 0, None, 0, True, True)
+        ds[0x00280106] = elem
+        out = correct_ambiguous_vr_element(elem, ds, True)
+        assert type(out) == DataElement
+        assert out.VR == 'US'
```

Comment:
I prefer `isinstance` for type checking, as it does exactly the same thing but allows for possibility of future subclasses or mixed classes.  I personally think it is more readable too, and is the generally recommended python practice.

However, it is unlikely to ever come up for DataElement, so can leave if you want to.

Reply:
Ah, I just copied another test, I'll fix it up.